packages/core/types/src/types/core/plugins/index.ts

```diff
@@ -14,3 +17,19 @@ export type IsEnabled<
       >
     : false
   : false;
+
+export type LoadedPlugin = {
+  config: {
```

Comment:
Oops, we merged plugin config types earlier this week, can you update from main so we can see if there's any overlap?